changelog.txt

```diff
@@ -1,3 +1,50 @@
+13.0-RC1 (2022-12-30)
+------------------------------------------------------------------------
+Feature: Slope-aware and roadtype-specific one-way sprites (#10282)
```

Comment:
This should have [NewGRF] probably.

I had renamed the PR/commit before merging to 'Add:' because 'Feature:' seems ott for something few players will notice.